ci/run_easymesh_cert.sh

```diff
@@ -54,29 +56,47 @@ is_prplmesh_device() {
     return 1
 }
 
+deploy_dummy_prplmesh() {
+    info "Deploy prplmesh locally to /opt/prplMesh"
+    rm -rf /opt/prplMesh
+    mkdir -p /opt/prplMesh
+    if ! mv "$rootdir/build/install" /opt/prplMesh ; then
```

Comment:
Err, won't this put the installation under /opt/prplMesh/install instead of under /opt/prplMesh?

More importantly, however: AFAIR, prplMesh isn't relocatable. When you move it somewhere else, some things won't work anymore. For example configuration file paths are hardcoded. So if we want to install it there, we should also build with CMAKE_INSTALL_PREFIX=/opt/prplMesh (and then install with DESTDIR=...). And that is a much bigger change I'm afraid...

Reply:
Indeed that's annoying...

Unless you see an issue with that, I'll update the CI configuration to always build with the prefix set to `/opt/prplMesh`.

Reply:
> Unless you see an issue with that, I'll update the CI configuration to always build with the prefix set to `/opt/prplMesh`.

I see an issue with that! test_flows.py assumes that it's installed in rootdir/build/install... So, if we move it to /opt/prplMesh (which is a good thing IMO), we should also change test_flows.py to start with a deployment that puts it there, and change maptools.py to use DESTDIR during installation. So it's quite a big change...

Reply:
Can't we do the deployment outside of test_flows.py and update it to point to /opt/prplMesh?